layers/meta-resin-allwinner/recipes-kernel/linux/linux-mainline_4.17.3.bb

```diff
@@ -24,4 +24,5 @@ SRC_URI = "https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-${PV}.tar.xz \
 	   file://don-t-default-activate-regdb.patch \
 	   file://0001-Enable-i2c0-on-OrangePi-Plus2-board.patch \
 	   file://0001-Revert-random-fix-crng_ready-test.patch \
+	   file://add-OrangePi-Zero-Wifi.patch \
```

Comment:
If you remove the kernel patch because we will move to thud branch shortly, this should also be removed

Reply:
Done :)